config.gradle

```diff
@@ -9,7 +9,7 @@ nativeUtils {
             wpiVersion = "2024.+"
             opencvYear = "frc2023"
             googleTestYear = "frc2023"
-            niLibVersion = "2023.3.0"
+            niLibVersion = "2024.+"

```

Comment:
We don't want + here, that risks breaking when the NI libraries change.  This should simply be the latest published 2024 version.

Reply:
(Which is 2024.1.1)